optaplanner-docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc

```diff
@@ -4,9 +4,7 @@
 :imagesdir: ../..
 :sectnums:
 :icons: font
-:quickstartsCloneUrl: https://github.com/kiegroup/optaplanner-quickstarts
-:quickstartsArchiveUrl: https://www.optaplanner.org/download/download.html
-:helloWorldJavaQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/hello-world
+include::../../_attributes.adoc[]
```

Comment:
The file must start with `_`, see https://docs.antora.org/antora/2.3/standard-directories/#hidden-files for an explanation.

Reply:
antora doesn't support ignores in their yml config? Not that I mind the _ so much.
good to merge as is.

Reply:
I haven't found such an option and would rather avoid it anyway; imagine maintaining a list of paths in a .yml file